api/v1/openapi.yaml

```diff
@@ -0,0 +1,953 @@
+      logstash:
+        description: Status of logstash integration
+        "$ref": "#/definitions/Status"
+      ipam:
+        description: Status of IP address management
+        "$ref": "#/definitions/IPAMStatus"
+  Status:
+    description: Individual status of a daemon component
+    type: object
+    properties:
+      state:
+        type: string
+        description: Overal state of component
+        enum:
+        - Ok
+        - Warning
+        - Failure
+        - Disabled
+      msg:
+        type: string
+        description: Human readable status/error/warning message
+  IPAMStatus:
+    description: Health and status information of daemon
+    properties:
+      ipv4:
+        type: array
+        items:
+          type: string
+      ipv6:
+        type: array
+        items:
+          type: string
+  DaemonConfigurationResponse:
+    description: |
+      Response to a daemon configuration request. Contains the adddresing
+      information and configuration settings.
+    type: object
+    properties:
+      addressing:
+        description: Addressing information of node
+        "$ref": "#/definitions/NodeAddressing"
+      configuration:
+        description: Configuration settings
+        "$ref": "#/definitions/Configuration"
+  Configuration:
+    description: |
+      General purpose structure to hold configuration for the daemon,
+      endpoints, etc. Split into a mutable and immutable part to specify
+      which settings can be modified.
+    type: object
+    properties:
+      mutable:
+        description: Changeable configuration
+        "$ref": "#/definitions/ConfigurationMap"
+      immutable:
+        description: Immutable configuration (read-only)
+        "$ref": "#/definitions/ConfigurationMap"
+  ConfigurationMap:
+    description: |
+      Map of configuration key value pairs.
+    type: object
+    additionalProperties:
+      type: string
+  NodeAddressing:
+    description: Addressing information of a node
+    type: object
+    properties:
+      ipv6:
+        "$ref": "#/definitions/NodeAddressingElement"
+      ipv4:
+        "$ref": "#/definitions/NodeAddressingElement"
+  NodeAddressingElement:
+    description: Addressing information element
+    type: object
+    properties:
+      enabled:
+        description: Address family is enabled
+        type: boolean
+      ip:
+        description: IP address of node
+        type: string
+      alloc-range:
+        description: Gateway address of containers
+        type: string
+  PolicyTree:
+    description: Policy tree or subtree
+    type: string
+  PolicyTraceResult:
+    description: Response to a policy resolution process
+    type: object
+    properties:
+      verdict:
+        type: string
+      log:
+        type: string
+  IdentityContext:
+    description: Context describing a pair of source and destination identies
+    type: object
+    properties:
+      from:
+        "$ref": "#/definitions/Labels"
```

Comment:
Does this still allows having a set of labels or an identity context? Should we extend this to allow a query to be performed against a source endpoint and a destination endpoint? or source container ID and a destination container ID?